external/builder/babel-plugin-pdfjs-preprocessor.mjs

```diff
@@ -170,6 +170,21 @@ function babelPluginPDFJSPreprocessor(babel, ctx) {
           );
           path.replaceWith(t.inherits(t.valueToNode(result), path.node));
         }
+
+        if (t.isIdentifier(node.callee, { name: "__non_webpack_import__" })) {
+          if (node.arguments.length !== 1) {
+            throw new Error("Invalid `__non_webpack_import__` usage.");
+          }
+          // Replace it with a standard `import`-call...
+          node.callee.name = "import";
+          // ... and ensure that Webpack will leave it alone.
+          node.arguments[0].leadingComments = [
+            {
+              type: "CommentBlock",
+              value: "webpackIgnore: true",
+            },
+          ];
```

Comment:
Instead of changing the identifier name to `import`, you should use the `ImportExpression` node:
```js
const source = node.arguments[0];
source.leadingComments = [
  {
    type: "CommentBlock",
    value: "webpackIgnore: true",
  },
];
path.replaceWith(t.importExpression(source));
```

because `import` is not a valid identifier. It currently happens to work in this case, but there is no guarantee that it will keep working with the other plugins (for example those in preset-env).